Model/Api/ShippingMethods.php

```diff
@@ -680,7 +680,7 @@ public function getShippingOptions($quote, $addressData)
 
             $diff = CurrencyUtils::toMinorWithoutRounding($cost, "USD") - $roundedCost;
 
-            $taxAmount = $this->cartHelper->getRoundAmount($shippingAddress->getTaxAmount() + $diff / 100);
+            $taxAmount = round(CurrencyUtils::toMinorWithoutRounding($shippingAddress->getTaxAmount(), "USD") + $diff);
```

Comment:
I slightly modified this but logic should be more or less same.
previously it was
`round(getTaxAmount() + diff / 100) * 100)`
now it is
`round(getTaxAmount() * 100 + diff)`

Reply:
was it 
`round((getTaxAmount() + diff / 100) * 100)` ? or was it 
`round(getTaxAmount() + diff / 100) * 100;`

just confirming so we dont get rounding error

Reply:
sorry it was `round((getTaxAmount() + diff / 100) * 100)` (the former)